langchain4j-anthropic/src/main/java/dev/langchain4j/model/anthropic/AnthropicStreamingData.java

```diff
@@ -0,0 +1,21 @@
+package dev.langchain4j.model.anthropic;
+
+class AnthropicStreamingData {
```

Comment:
nit: maybe would it be worth to extend `AnthropicCreateMessageResponse`

Reply:
I would keep them apart to avoid potential conflicts, they are used in 2 different ways, so it does not make much sense to couple them together